src/tests/test_api.rs

```diff
@@ -34,5 +34,6 @@ pub trait TestApi {
     fn foreign_key(&mut self, parent_table: &str, parent_column: &str, child_column: &str) -> String;
     fn autogen_id(&self, name: &str) -> String;
     fn conn(&self) -> &crate::single::Quaint;
+    async fn new_conn(&self) -> crate::Result<crate::single::Quaint>;
```

Comment:
Even though it's a test setup, the naming here might be a bit better. It's not super clear what's the difference between this and `conn`...

Reply:
Renamed it to `create_additional_connection`